lib/typescript/typescript_transpiler.js

```diff
@@ -0,0 +1,33 @@
+import ts from 'typescript';
+import fs from 'fs';
+import path from 'path';
+
+const TYPESCRIPT_COMPILER_OPTIONS = {
+    compilerOptions: {
```

Comment:
What is the difference between these compiler options and the compiler options defined in the separate file at the root level?